### ISSUE TITLE
Possible conflict with Rank Math and/or The Plus Addons

### ISSUE THREAD

**Issue by over-engineer** [over-engineer/lottie-for-elementor#6]
Possible conflict with [“WordPress SEO Plugin – Rank Math”](https://wordpress.org/plugins/seo-by-rank-math/) and/or [“The Plus Addons for Elementor Page Builder”](https://wordpress.org/plugins/the-plus-addons-for-elementor-page-builder/)

Rank Math support reported the following error message:

```
frontend.js?ver=1.0.0:144 Uncaught ReferenceError: elementorFrontend is not defined
    at HTMLDocument.<anonymous> (frontend.js?ver=1.0.0:144)
    at i (load-scripts.php?c=0&load[chunk_0]=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-widget,jquery-ui-mouse,shortcode,utils,wp-util,wp-backbo&load[chunk_1]=ne,media-models,moxiejs,plupload,wp-plupload,jquery-ui-sortable&ver=5.4:4)
    at Object.fireWith [as resolveWith] (load-scripts.php?c=0&load[chunk_0]=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-widget,jquery-ui-mouse,shortcode,utils,wp-util,wp-backbo&load[chunk_1]=ne,media-models,moxiejs,plupload,wp-plupload,jquery-ui-sortable&ver=5.4:4)
    at Function.ready (load-scripts.php?c=0&load[chunk_0]=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-widget,jquery-ui-mouse,shortcode,utils,wp-util,wp-backbo&load[chunk_1]=ne,media-models,moxiejs,plupload,wp-plupload,jquery-ui-sortable&ver=5.4:4)
    at HTMLDocument.J (load-scripts.php?c=0&load[chunk_0]=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-widget,jquery-ui-mouse,shortcode,utils,wp-util,wp-backbo&load[chunk_1]=ne,media-models,moxiejs,plupload,wp-plupload,jquery-ui-sortable&ver=5.4:4)
```

References
---

* [“Conflict with RankMath” — WordPress Plugin Support](https://wordpress.org/support/topic/conflict-with-rankmath/)
* [“SEO tab not working for all pages” — Rank Math support ticket](https://support.rankmath.com/reply/43298/)

---

<sub>**Edit:** Updated references and added the error message.</sub>

**Comment by over-engineer** [comment]
Just released version 1.1.1, which introduces a _possible_ fix for this issue.

**Comment by over-engineer** [comment]
Confirmed it was **resolved in 1.1.1**
